en/docs/get-started/quick-start-guide.md

```diff
@@ -633,22 +634,24 @@ Pickup Manager applications using WSO2 Identity Server.
         2.  Navigate back to the Pickup Dispatch application and sign in
             using the new user credentials.
 
-        ![qsg-sso-login-credentials](../assets/img/getting-started/qsg-sso-login-credentials.png)
+        ![qsg-sso-login-credentials](../../../assets/img/get-started/qsg-sso-login-credentials.png)
 
         Note that the Dispatch home screen appears.
 
-        ![qsg-sso-dispatch-home](../assets/img/getting-started/qsg-sso-dispatch-home.png)
+        ![qsg-sso-dispatch-home](../../../assets/img/get-started/qsg-sso-dispatch-home.png)
 
         You have just self-signed up to a Pickup web
         application.
 
 9.  Next, in order to try out other scenarios, navigate back to the
     command prompt where you ran the Quick Start sample and enter `y` to clean the setup.
-    ![qsg-sso-cleanup](../assets/img/getting-started/qsg-sso-cleanup.png)
+    ![qsg-sso-cleanup](../../../assets/img/get-started/qsg-sso-cleanup.png)
 
-----------------
-### Workflows
-##### Problem Scenario
+---
+
+## Workflows
```

Comment:
better to comment this section as workflow requests can't be approved with the management console or user-portal